code/obj/machinery/shield_generators/shield_generator.dm

```diff
@@ -136,8 +136,10 @@
 			user.show_text("You flip the switch on [src].")
```

Comment:
This should be a visible message I think.

Reply:
it is, however, it only says you flip the switch on the machine, not fliping ON the machine

Reply:
he means it should use the proc atmo/proc/visible_message()

Reply:
visible message proc will also allow the blind to still get a message of some kind

Reply:
Ah, thanks for clarifying ill change that immediatly